tests/mgbench/client.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Memgraph Ltd.
+// Copyright 2023 Memgraph Ltd.
```

Comment:
Please revert this if you haven't done any other changes. If you cannot because of the pre-commit hook, then it is fine as is.

The same applies to other files also.